### PR TITLE
Next Release - v0.5.2

### PR DESCRIPTION
**Pull request by srivarra**
**If you haven't already, please read through our [contributing guidelines](https://ark-analysis.readthedocs.io/en/latest/_rtd/contributing.html) before opening your PR**

**What is the purpose of this PR?**

Changes needed for a new release. Also, an opportunity to test the release workflow, and find any straggling issues.

**How did you implement your changes**

Added the CI workflow in the README.md

**Remaining issues**

- [x] Update Develop Docs
- [x] Test PyPI working, see the following [actions' run](https://github.com/angelolab/ark-analysis/actions/runs/4188583810)
- [x] Make sure the Docker builds and uploads successfully [here](https://hub.docker.com/layers/angelolab/ark-analysis/releases-v0.5.2/images/sha256-6d6dade537f700254e841ff8b554a292b4e1bcd6ecb263aa1d59a1450f3efb2a?context=explore).
- [x] Make sure the Docker runs

### CODE REVIEW

**Comment by ngreenwald** [comment]
Will you also update the developer docs section? I think some of the stuff about M1 macs is no longer relevant. The instructions for installing also still assumes there's a requirements.txt file, which I think now is gone in favor of the setup file. Finally, will you remove the section about making a new release, since all of that is in the comp wiki?

**Comment by alex-l-kong** [comment]
@ngreenwald do we want this to be the release that Candace can show to the reviewers? If so, we'll need to wait until we've merged in her requested fixes before merging this PR in.

**Comment by ngreenwald** [comment]
Nope, we still have more changes for her project that need to get in before we can make a release for the paper

**Comment by alex-l-kong** [comment]
> Nope, we still have more changes for her project that need to get in before we can make a release for the paper

Sounds good, just checking.

File: docs/_rtd/development.md
Comment (ngreenwald) [inline docs/_rtd/development.md:55]: This doesn't work anymore, right?

**Review by ngreenwald — CHANGES_REQUESTED** [review]
Looks good, just a question about developer environments

File: docs/_rtd/development.md
Comment (srivarra) [reply docs/_rtd/development.md:55]: It's not needed anymore. The Apple silicon instructions are now the same as with any other OS / architecture.

**Review by ngreenwald — APPROVED** [review]
Sweet, looks good

**Pull request merged**
